.github/workflows/ci.yml

```diff
@@ -1,9 +1,15 @@
 name: CI
 
-on: [
-    push,
-    pull_request
-]
+on:
+  pull_request:
```

Comment:
Can the trailing colon be removed?

Reply:
We can probably fix those up by script/regex automatically as follow up for all repos if we find that out.